common/src/thrift/file_system_master.thrift

```diff
@@ -42,6 +43,16 @@ struct MkdirTOptions {
   3: optional bool allowExists
 }
 
+struct PersistCommand {
+  1: common.CommandType commandType
```

Comment:
PersistCommand is used for worker heartbeat between file system master and worker. So in the future I can see other types of command can be passed along. So I think this command will be renamed later to something more general.